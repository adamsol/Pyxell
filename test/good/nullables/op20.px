
a: [String?]?
print a?[0:5], a?[:]

a = [for c in 'a'..'e' yield c]
a![-1] = a![0] = null
print a?[3:]
print [for b in a![2::-1] yield b?[::5]]
