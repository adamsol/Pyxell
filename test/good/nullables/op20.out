null null
["d", null]
["c", "b", null]
